Description: Add Keywords= field to mate-calc.desktop file
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/data/mate-calc.desktop.in
+++ b/data/mate-calc.desktop.in
@@ -7,6 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
+_Keywords=calculator,MATE,scientifc,arithmetic,financial,calculations
 Categories=GTK;Utility;Calculator;
 X-MATE-DocPath=mate-calc/mate-calc.xml
 X-MATE-Bugzilla-Bugzilla=MATE
